src/components/_common/contact-option/ChatOption.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Alert, BodyLong, Heading } from '@navikt/ds-react';
+import { ChatData } from 'types/component-props/parts/contact-option';
+import { translator } from 'translations';
+import { usePageConfig } from 'store/hooks/usePageConfig';
+import { LenkeBase } from 'components/_common/lenke/LenkeBase';
+import { classNames } from 'utils/classnames';
+import { AnalyticsEvents } from 'utils/amplitude';
+import { useLayoutConfig } from '../../layouts/useLayoutConfig';
+import { openChatbot } from '@navikt/nav-dekoratoren-moduler';
+import { ParsedHtml } from '../parsed-html/ParsedHtml';
+
+import style from './ContactOption.module.scss';
+
+export const ChatOption = (props: ChatData) => {
+    const { ingress, title, alertText } = props;
+    const { language } = usePageConfig();
+    const { layoutConfig } = useLayoutConfig();
+    const getTranslations = translator('contactPoint', language);
+
+    const getTitle = () => {
+        const dictionaryTitle = getTranslations('chat').title;
+        return title || dictionaryTitle;
+    };
+
+    const getIngress = () => {
+        const dictionaryIngress = getTranslations('chat').ingress;
+        return ingress || dictionaryIngress;
+    };
+
+    return (
+        <div className={style.contactOption}>
+            <LenkeBase
+                onClick={(e) => {
+                    e.preventDefault();
+                    openChatbot();
+                }}
+                href="#"
+                analyticsEvent={AnalyticsEvents.CHAT_OPEN}
+                analyticsLinkGroup={layoutConfig.title}
+                analyticsComponent={'Kontakt-oss kanal'}
+                className={style.link}
+            >
```

Comment:
Kan `<button>` være et alternativ? (Siden klikk på `<a href="#">` er overstyrt)

Reply:
Ja, da måtte vi ha overridet den som en anchor siden button vel semantisk ikke kan inneholde divs. Så for å ikke ødelegge analytics-historikk så tenkte jeg bare å beholde LenkeBase slik den var brukt på første versjon av ChatOption.